namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-maat-test/resources/irsa.tf

```
@@ -0,0 +1,39 @@
+module "irsa" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=2.0.0"
+
+  # EKS configuration
+  eks_cluster_name = var.eks_cluster_name
+
+  # IRSA configuration
+  service_account_name = "sa-laa-crime-apps-test"
+  namespace            = var.namespace # this is also used as a tag
+
+  # Attach the approprate policies using a key => value map
+  # If you're using Cloud Platform provided modules (e.g. SNS, S3), these
+  # provide an output called `irsa_policy_arn` that can be used.
+  role_policy_arns = {
+  s3  = module.s3.irsa_policy_arn
```

Concourse is complaining that there isnt a module resource called `s3` defined, the resource I think you are trying to call in s3.tf is called `s3_bucket`. Making the following update to the code should hopefully fix it! 
```suggestion
  s3  = module.s3_bucket.irsa_policy_arn
```